test/verify/check-static-login

```diff
@@ -719,6 +721,22 @@ account    required     pam_succeed_if.so user ingroup %s""" % m.get_admin_group
         self.allow_journal_messages(".*[aA]ccount .*locked due to .* failed logins.*")
         self.allow_journal_messages(".*minutes left to unlock.*")
 
+    @skipImage("root logins disabled by default with ssh", "fedora-coreos")
+    def testPamAccess(self):
+        b = self.browser
+
+        # root login works by default
+        self.login_and_go(user="root")
+        b.logout()
+
+        # disable root login with pam_access
+        self.write_file("/etc/security/access.conf", "- : root : ALL\n", append=True)
+        self.sed_file("1 aaccount required pam_access.so", "/etc/pam.d/cockpit")
```

Comment:
`a` :)

Reply:
How do you mean? Is there a way to make this more readable and not look like a typo?

Reply:
No. I'm just happy to see you using `a` here, after our discussion the other day :)